packages/core/src/components/ui/Newsletter/NewsletterAddendum.tsx

```diff
@@ -13,6 +13,8 @@
  *
  * This is a limitation not only for this component, but for every native & custom component that makes use of Rich Text.
  */
+
+// @ts-ignore motivation: cannot find the draftjs-to-html package types.
```

Comment:
The comment above says that this library will be removed soon. Her types are missing. I could try to install or create a d.ts file. However, the idea is that the library is removed